packages/oscal-react-library/src/components/OSCALCatalogGroup.tsx

```diff
@@ -159,96 +143,29 @@ const CollapsibleListItem: React.FC<CollapsibleListItemProps> = (props) => {
     previousHandledFragment,
     setPreviousHandledFragment,
   ]);
-
-  return (
-    <StyledListItemPaper>
-      <StyledListItem onClick={handleClick}>
-        <ListItemText primary={itemText} />
-        {isOpen ? <ExpandLess /> : <ExpandMore />}
-      </StyledListItem>
-      <Collapse
-        in={isOpen}
-        timeout="auto"
-        onEntered={() => !group && setListItemOpened(true)}
-        unmountOnExit
-      >
-        <StyledControlDescriptionWrapper>{children}</StyledControlDescriptionWrapper>
-      </Collapse>
-    </StyledListItemPaper>
-  );
-};
-
-interface WithdrawnListItemProps extends CatalogGroupFragmentProps {
-  /**
-   * Children underneath the list item - either a control/group or control/group
-   */
-  readonly children: ReactNode;
-  /**
-   * The current fragment being handled
-   */
-  readonly urlFragment?: string;
-  /**
-   * Identification used for a list
-   */
-  readonly listId: string;
-  /**
-   * The end of a fragment which starts with the current group/control being handled
-   */
-  readonly fragmentSuffix: string | undefined;
-}
-
-/**
- * Creates a top-level withdrawn control list item.
- *
- * @returns A top-level withdrawn control list item component
- */
-const WithdrawnListItem: React.FC<WithdrawnListItemProps> = (props) => {
-  const {
-    children,
-    urlFragment,
-    listId,
-    fragmentSuffix,
-    previousHandledFragment,
-    setPreviousHandledFragment,
-  } = props;
-  useEffect(() => {
-    if (
-      urlFragment &&
-      isMatchingListItem(urlFragment, previousHandledFragment, fragmentSuffix, listId)
-    ) {
-      const elementWithFragment = document.getElementById(urlFragment);
-      elementWithFragment?.scrollIntoView?.({ behavior: "smooth" });
-
-      if (fragmentSuffix && fragmentSuffix.split("/").length <= 1) {
-        setPreviousHandledFragment(urlFragment);
-      }
-    }
-  }, [urlFragment, listId, fragmentSuffix, previousHandledFragment, setPreviousHandledFragment]);
-
   return (
-    <StyledListItemPaper>
-      <StyledListItem>{children}</StyledListItem>
-    </StyledListItemPaper>
+    <Accordion
+      expanded={isOpen}
+      onChange={handleClick}
+      TransitionProps={{
+        onEntered: () => !group && setListItemOpened(true),
+        unmountOnExit: true,
+        timeout: 0,
```

Comment:
This is what we're using to prevent the long transitions. If you want to have animations but faster, then increase `timeout`. It is `0` because as-is on `develop` today, if you open AC-3 and then close it, you have to wait for the animation to close things _below your viewport_ before you see any feedback that you clicked the button.

Maybe 0 is too jarring but suggest a value.

Reply:
I think 0 is fine for now. It is a pretty simple thing to change later if we want to.

Reply:
I changed this to `500`